### ISSUE TITLE
Additional Article Options [Feature Request]

### ISSUE THREAD

**Issue by versavius** [opensupports/opensupports#353]
The article section is great but we are lacking some options which we should be able to set for each article:

Visible By (who can view the article)
- Public (everyone)
- Visible to Logged In Users Only

Article Status:
- Draft (only visible to staff)
- Live (visible to customers)
- Archived (visible to staff only)

Topic Status: (a topic status should hide all articles within that topic)
- Visible to Public
- Visible to Logged In Users Only
- Archived (visible to staff only)

This should make the articles section much more powerful and allows you to have articles only visible to logged in users which might be useful if you provide a service and the information you provide is of a premium quality.

**Comment by nickbe** [comment]
I think restricting articles to staff or user is fine, although I don't think there should be a difference between logged in or not. But maybe articles could simply be bound to departments. This way all department restrictions would also apply to articles. But it would still keep the system simple enough.

**Comment by versavius** [comment]
Yes that would make sense, the only reason to force users to be logged in is for privacy the business may have certain articles to do with HR policy / sales policy etc that you would not want everyone to view you would only want logged in users to access this provides additional privacy and allows you to use the article section as a true wiki for internal and external usage.

Confluence is a good example of this where articles can be restricted by logged in users only
